### ISSUE TITLE
Add pak version 11 support

### ISSUE THREAD

**Issue by wmtorode** [panzi/rust-u4pak#3]
MechWarrior 5 Mercenaries uses version 11 of the pak file, which is different enough that it cant be used with this tool.

at a quick glance the magic in the footer is now farther from the end of the file so this tool can't read it

**Comment by panzi** [comment]
Does `u4pak info` (or `u4pak --ignore-magic`) say that everything is 0? Then I don't know what to do anyway.
However, I don't own that game and I wouldn't have time to look at it at the moment anyway.

**Comment by cooperdk** [comment]
I just tried with another game, and yes, everything is reported with a value of 0.

**Comment by yvt** [comment]
[This game's demo version](https://store.steampowered.com/app/482400/System_Shock/) uses version 11. `u4pak info --ignore-magic` reports 0 for everything.

**Comment by Xenira** [comment]
I know this is older, but I just want to give an update on this issue.

1. Pak files are currently only supported till v7. Version >= 8 introduced changes to the footer causing the all 0. I already created a PR for up to version 9 (#7).
2. Version >= 10 introduced a secondary index containing encoded file paths. This will cause pak files to appear empty with the canges introduced in v8 support. I just finished a first partial implementation for v11 support (only reading/extracting, no signature checking). Could be some time till this is merged though.
